pos_partner_location_abstract/models/pos_config.py

```diff
@@ -0,0 +1,25 @@
+from odoo import api, fields, models
+
+
+class PosConfig(models.Model):
+    _inherit = "pos.config"
+
+    geolocalize_tech_name = fields.Char(compute="_compute_geolocalize")
```

Comment:
If I understand this field correctly, this should be a Selection field, don't you think ? (openstreetmap / googlemap). don't you think ?

Reply:
> If I understand this field correctly, this should be a Selection field, don't you think ? (openstreetmap / googlemap). don't you think ?

I thinks it's not better idea. This field is compute by `Geo Provider` configuration parameter. Provider we can update or add new what raise an error, that is why this field is just Char.